src/model.rs

```diff
@@ -64,6 +58,8 @@ pub struct Pod {
     output_stream_map: BTreeMap<String, StreamInfo>,
     recommended_cpus: f32,
     recommended_memory: u64,
+    /// Metadata that doesn't affect reproducibility.
+    pub annotation: Option<Annotation>,
```

Comment:
I am not so sure of moving annotation to a lower line

Reply:
I was trying to follow a common pattern of pushing optional args (ones with a default (`None`)) towards the end of constructors (`new`). This is quite minor and I don't feel super strongly either way. Mainly just think we should align in general so we are consistent.